src/scripts/executeFork.js

```diff
@@ -0,0 +1,63 @@
+/* eslint-disable no-await-in-loop, no-use-before-define, no-lonely-if, import/no-dynamic-require, global-require */
+/* eslint-disable no-console, no-inner-declarations, no-undef, import/no-unresolved, no-restricted-syntax */
+const path = require('path');
+const { ethers } = require('hardhat');
+require('dotenv').config({ path: path.resolve(__dirname, '../.env') });
+
+async function main() {
+    const args = process.argv.slice(2);
+    const deploymentName = args[0];
+    const deployParameters = require(`../../deployments/${deploymentName}/deploy_parameters.json`);
+    const deploymentOutput = require(`../../deployments/${deploymentName}/deploy_output.json`);
+
+    const mandatoryDeploymentOutput = [
```

Comment:
NBD but I think you could use `common.verifyDeploymentParameters` for this, should save a few lines